include/vulkan/utility/vk_format_utils.h

```diff
@@ -2036,10 +2043,14 @@ inline VkExtent3D vkuFormatTexelBlockExtent(VkFormat format) { return vkuGetForm
 
 inline enum VKU_FORMAT_COMPATIBILITY_CLASS vkuFormatCompatibilityClass(VkFormat format) { return vkuGetFormatInfo(format).compatibility; }
 
+inline uint32_t vkuFormatTexelBlockSize(VkFormat format) { return vkuGetFormatInfo(format).texel_block_size; }
+
+// Deprecated - Use vkuFormatTexelBlockSize
 inline uint32_t vkuFormatElementSize(VkFormat format) {
     return vkuFormatElementSizeWithAspect(format, VK_IMAGE_ASPECT_COLOR_BIT);
 }
 
+// Deprecated - Use vkuFormatTexelBlockSize
 inline uint32_t vkuFormatElementSizeWithAspect(VkFormat format, VkImageAspectFlagBits aspectMask) {
     // Depth/Stencil aspect have separate helper functions
     if (aspectMask & VK_IMAGE_ASPECT_STENCIL_BIT) {
```

Comment:
Why not just remove the functions? Deprecation comments are... unlikely to be noticed and just delay using the correct function. 

I'm fine with just yeeting the functions to speed up the process. Maybe add a comment to the new functions with "used to be called X".

Reply:
`vkuFormatElementSizeWithAspect` is used in `vkuFormatTexelSizeWithAspect` still... want to remove `vkuFormatTexelSizeWithAspect` as well, but need a replacement for that, so all of them gone soon

Reply:
I'm fine with just deprecating them, but wanted to nip "forgotten deprecation notice" in the bud, IE don't forget that we wanted to remove this. Cause if we forget to remove it, then why not just keep it working long term...

Reply:
I guess asking for a "trust me bro" here, this will be gone by end of week (need working group time tomorrow to clarify a few last things before hard removing)

Reply:
That's more than enough for me!

And yeah I failed to notice how one of the deprecated functions isn't just renamed, but removed 'functionality'.